core/tests/build_sources/templates/base.html

```diff
@@ -111,7 +111,6 @@
       </div>
 
       <div class="oppia-base-container"
-           ng-class="{'oppia-sidebar-menu-open': isSidebarShown(), 'oppia-sidebar-menu-closed': !isSidebarShown()}"
```

Comment:
Hm, why isn't isSidebarshown() changing it's value like it does on other pages? The appropirate class should be added automatically based on the value for isSidebarShown(). This could be because isSidebarShown() in the splash page is not in the right scope. The right fix for this would be to ensure isSidebarShown() returns the right value.

Reply:
![sidebar-open](https://user-images.githubusercontent.com/11008603/61378921-115ee980-a8c4-11e9-9d55-8f4fb0bb02ac.png)
Notice how the `oppia-sidebar-menu-open` class is added when the menu is open in the library page.

Reply:
Attaching the SidebarStatusService.isSidebarShown to $scope in the BaseContentDirective.ts seems to fix this. Something like:

```suggestion
      controller: [
        '$scope'
        function($scope) {
          $scope.isSidebarShown = SidebarStatusService.isSidebarShown;
        }
      ]
```

But I'd defer to the reviewers regarding this fix.
/cc @seanlip @DubeySandeep

Reply:
Hey @kevintab95 there was some issue happening there with the classes so that's why I  removed that classes part and applied the same property using javascript. It's working fine now.

Reply:
@Showtim3, Let's avoid DOM manipulation in the controller layer. Try using the controller for "business logic", can you find a way to do this without doing dom manipulation in the controller?